lvsfunc/comparison.py

```diff
@@ -605,8 +609,13 @@ def diff(*clips: vs.VideoNode,
     if (clips and len(clips) != 2) or (namedclips and len(namedclips) != 2):
         raise CustomValueError("Must pass exactly 2 `clips` or `namedclips`!", diff)
 
-    if not 1 <= thr < 128:
-        raise CustomValueError(f"`thr` must be between 1 and 128, not {thr}!", diff)
+    if not thr < 128:
+        raise CustomValueError(f"`thr` must be below 128!", diff)
+    
+    if also_check_avg_thr == None and thr > 1:
+        also_check_avg_thr = (128 - thr) * 0.000046875 + 0.0105
```

Comment:
Magic numbers?

Reply:
sorry, I don't understand. note that i explained below that these values were derived experimentally.